content/en/docs/faq/troubleshooting.md

```diff
@@ -128,3 +128,45 @@ For example:
 helm list -v 6
 ```
 
+### Tiller installations stopped working and access is denied
+
+Helm releases used to be available from [https://storage.googleapis.com/kubernetes-helm/](https://storage.googleapis.com/kubernetes-helm/). As explained in ["Announcing get.helm.sh"](https://helm.sh/blog/get-helm-sh/), the official location changed in June 2019.
+
+The Tiller image [legacy location](https://gcr.io/kubernetes-helm/tiller) remained available until August 2021, and images began being removed at that time. [GitHub Container Registry](https://github.com/orgs/helm/packages/container/package/tiller) makes all the old Tiller images available.
+
+You may see an error with a command-line install:
+```
+$ helm tiller install
```

Comment:
`helm tiller` is a third party plugin. This would be the first time we are documenting it. I’d suggest documenting the behaviour of `helm init` anyways as that’s the recommended solution to initialize tiller.